### PR TITLE
feat(blocks): embed html fullscreen toolbar

### PR DESCRIPTION
**Pull request by golok727**
Closes: [AFF-1090](https://linear.app/affine-design/issue/AFF-1090/add-toolbar-for-fullscreen-html-block)

https://github.com/toeverything/blocksuite/assets/123532141/b50269ac-dd3a-4b4a-bbc6-d6926e979d55

### CODE REVIEW

**Comment by golok727** [comment]
* **#7138** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7138?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @golok727 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+  override render() {
+    const hideToolbar = !this._popperVisible && this.autoHideToolbar;
+
+    return html`<div
+      data-auto-hide=${hideToolbar}
+      class="toolbar-toggle-control"
+    >
+      <div class="fullscreen-toolbar-container">
+        <div class="left">
```
Comment (fundon) [inline packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts:178]: May not need left or right to wrap buttons

File: packages/blocks/src/embed-html-block/styles.ts [hunk 2]
```diff
@@ -59,30 +59,22 @@ export const styles = css`
...
-    cursor: pointer;
-    color: var(--affine-white);
-    border: 1px solid var(--affine-black-10);
-    background: var(--affine-primary-color);
-  }
-  .embed-html-block-iframe-wrapper:fullscreen > .iframe-tip {
-    display: flex;
+
+  .embed-html-block-iframe-wrapper > embed-html-fullscreen-toolbar {
+    display: none;
```
Comment (fundon) [inline packages/blocks/src/embed-html-block/styles.ts:74]: Maybe `visibility: visible` will be better.

File: packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+      justify-content: space-between;
+      align-items: center;
+      gap: 12px;
+
+      padding: 0 20px;
+
+      height: 64px;
+
+      -webkit-user-select: none;
+      user-select: none;
```
Comment (fundon) [inline packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts:55]: It's not need here.

File: packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+    .fullscreen-toolbar-container .left,
+    .fullscreen-toolbar-container .right {
+      display: flex;
+      gap: 8px;
+    }
+
+    .fullscreen-toolbar-container icon-button svg {
+      width: 24px;
+      height: 24px;
```
Comment (fundon) [inline packages/blocks/src/embed-html-block/components/fullscreen-toolbar.ts:73]: icon-button has `width` and `height` properties.

**Review by fundon — CHANGES_REQUESTED** [review]

**Review by fundon — APPROVED** [review]

**Review by doodlewind — APPROVED** [review]
Thanks for the feat!

**Pull request merged**